Building of Windows installers failed for DOSBox-X 2023.05.01

### Describe the bug

Action "Windows installers" (windows-installers.yml) fails:
```
Waiting for DOSBox-X Windows builds to be uploaded to the latest release.
dosbox-x-vsbuild-XP-20230501152329.zip already exists (use `--clobber` to overwrite file or `--skip-existing` to skip file)
Error: Process completed with exit code 1.
```

### Steps to reproduce the behaviour

Create a new release of DOSBox-X.

### Expected behavior

_No response_

### What operating system(s) this bug have occurred on?

Github CI

### What version(s) of DOSBox-X have this bug?

2023.05.01

### Used configuration

_No response_

### Output log

_No response_

### Additional information

_No response_

### Have you checked that no similar bug report(s) exist?

- [X] I have searched and didn't find any similar bug report.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

--- a/.github/workflows/windows-installers.yml
+++ b/.github/workflows/windows-installers.yml
@@ -46,7 +46,7 @@ jobs:
           # Wait until all required DOSBox-X builds are available.
           DOSBOX_BUILDS_AVAILABLE=false
           while [ $DOSBOX_BUILDS_AVAILABLE != true ]; do
-            gh release download -p 'dosbox-x-mingw-win*.zip' -p 'dosbox-x-vsbuild-*.zip'
+            gh release download --skip-existing -p 'dosbox-x-mingw-win*.zip' -p 'dosbox-x-vsbuild-*.zip'
             # Rename the files so that the date in the file name matches the release date.
             DOSBOX_X_RELEASE_COMPACT=`echo ${DOSBOX_X_RELEASE} | sed -e "s/\.//g"`
             for file in `ls dosbox-x-*.zip`; do

--- a/.github/workflows/windows-installers.yml
+++ b/.github/workflows/windows-installers.yml
@@ -29,7 +29,8 @@ jobs:
         run: |
           # Wait long enough to make sure the new release has been created and the Windows binaries have been built.
           #MINUTES_WAIT=0
-          MINUTES_WAIT=45
+          #MINUTES_WAIT=45
+          MINUTES_WAIT=120
           for minutes in `seq $MINUTES_WAIT`; do
             sleep 60
             echo "Waiting: $minutes of $MINUTES_WAIT minutes"
@@ -60,7 +61,6 @@ jobs:
             if [ `ls dosbox-x-vsbuild-win64-*.zip 1>/dev/null 2>&1; echo $?` != 0 ]; then DOSBOX_BUILDS_AVAILABLE=false; fi
             if [ `ls dosbox-x-vsbuild-arm32-*.zip 1>/dev/null 2>&1; echo $?` != 0 ]; then DOSBOX_BUILDS_AVAILABLE=false; fi
             if [ `ls dosbox-x-vsbuild-arm64-*.zip 1>/dev/null 2>&1; echo $?` != 0 ]; then DOSBOX_BUILDS_AVAILABLE=false; fi
-            if [ `ls dosbox-x-mingw-win32-*.zip 1>/dev/null 2>&1; echo $?` != 0 ]; then DOSBOX_BUILDS_AVAILABLE=false; fi
             if [ `ls dosbox-x-mingw-win32-*.zip 1>/dev/null 2>&1; echo $?` != 0 ]; then DOSBOX_BUILDS_AVAILABLE=false; fi
             if [ `ls dosbox-x-mingw-win64-*.zip 1>/dev/null 2>&1; echo $?` != 0 ]; then DOSBOX_BUILDS_AVAILABLE=false; fi
             if [ $DOSBOX_BUILDS_AVAILABLE == false ]; then

